.circleci/config.yml

```diff
@@ -111,7 +111,12 @@ jobs:
           key: protocol-completed-build-{{ .Environment.CIRCLE_SHA1 }}
       - run:
           name: Run tests
-          command: yarn run test
+          command: |
+            cd ..
+            mkdir truffle_workaround
+            mv protocol truffle_workaround/
+            cd truffle_workaround/protocol
+            yarn run test
```

Comment:
Should we `rm -rf truffle_workaround` somewhere in ci or does circleci handle cleanup

Reply:
circleci handles the cleanup as far as I'm aware. We don't cache the results of this job for any other job to use, so I think we should be fine just moving stuff around without cleaning up.